lib/widgets/subscription_list.dart

```diff
@@ -215,30 +223,42 @@ class SubscriptionItem extends StatelessWidget {
           const SizedBox(width: 16),
           Padding(
             padding: const EdgeInsets.symmetric(vertical: 11),
-            child: Icon(size: 18, color: swatch.iconOnPlainBackground,
-              iconDataForStream(subscription))),
+            child: Opacity(
+              opacity: opacity,
+              child: Icon(size: 18, color: swatch.iconOnPlainBackground,
+                iconDataForStream(subscription)),
+            ),
+          ),
```

Comment:
nit: follow the existing style for formatting these close-parens
```suggestion
                iconDataForStream(subscription)))),
```